plugins/Dispersion/DispersionControls.h

```diff
@@ -0,0 +1,82 @@
+/*
+ * DispersionControls.h
+ *
+ * Copyright (c) 2023 Lost Robot <r94231/at/gmail/dot/com>
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+#ifndef LMMS_DISPERSION_CONTROLS_H
+#define LMMS_DISPERSION_CONTROLS_H
+
+#include "EffectControls.h"
+#include "DispersionControlDialog.h"
+
+namespace lmms
+{
+
+class DispersionEffect;
+
+namespace gui
+{
+class DispersionControlDialog;
+}
+
+
+class DispersionControls : public EffectControls
+{
+	Q_OBJECT
+public:
+	DispersionControls(DispersionEffect* effect);
+	~DispersionControls() override = default;
+
+	void saveSettings(QDomDocument & _doc, QDomElement & _parent) override;
+	void loadSettings(const QDomElement & _this) override;
```

Comment:
```suggestion
	void saveSettings(QDomDocument& doc, QDomElement& elem) override;
	void loadSettings(const QDomElement& elem) override;
```
I missed this in my initial review